### DISCUSSION TITLE
837 schemas

### DISCUSSION THREAD

**Discussion by themichaelsweeney** [xlate/staedi#182] [General]
Hey, just wanted to mention that I'm also working through the 837 schemas and I've been refining an implementation schema xml for 837P and 837I. If you'd like me to send that along, or if there's any other way I can contribute, please let me know.

Best, Mike

**Comment by MikeEdgar** [comment]
Hi @themichaelsweeney - thank you for the offer to contribute. As much as I would like to include the schemas here or in a separate repo, I have been avoiding it due to concerns around copyright and IP.

**Reply by saparv** [reply]
@themichaelsweeney - I am planning on using this library and wondering if I can get head start with schemas. Please let me know if you are willing to share.

**Comment by ibbybuilds** [comment]
@themichaelsweeney Can you please share these schemas with me? I am working on converting edi to csv and these schema would be very helpful. Thanks!
